spec/features/inject_method_feature_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'array'
+describe 'method works the same way as inject:' do
+
+  it 'with strings' do
+    array = ["inject", "is", "cool"]
+    expect(array.molliesinject{|sentence, word| sentence + word}).to eq "injectiscool"
```

Comment:
Line is too long. [86/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.